### ISSUE TITLE
[Feature Request] A way to stop the bot from triggering on a comment

### ISSUE THREAD

**Issue by SomewhatDamaged** [JeremyRuhland/saucebot-discord#24]
**Feature**: A way to stop the bot from triggering on a comment.

**Problem to solve**: Sometimes the bot responds in undesirable ways to posted links (tweets with a single image, etc).

**Suggested method**: Have an emoji that can be used in a message that stops the bot triggering.

Example: 
```
https://twitter.com/mcnostrilcom/status/1509347598045523973 🚫
```

**Comment by goopypanther** [comment]
Ah yes, this is a side effect of twitter not accurately reporting which tweets will generate embeds and which will not (the `possibly_sensitive` property only kind of corresponds to this but there are tweets with this property that still generate native discord embeds and tweets that do not have it that will not generate embeds).

There's currently a number of ways to stop the bot from activating but they also stop discord from generating native embeds as well (wrapping URLs in double pipes or angle brackets) so this won't really solve the problem.

I can add another ignore rule like you suggested but its unlikely most users would know the feature exists. Another major problem is that it isn't clear to users which tweets will cause this double-embedding issue to begin with, so they wouldn't know to include the bot-ignore command until it was already too late.

**Comment by SomewhatDamaged** [comment]
Yeah, the whole "random 404" thing is odd. I dev a Discord bot myself and while my production server (on DigitalOcean) is fine and can see all tweets, my dev environment on my home connection isn't, and suffers from the same problem as Discord's own embed unfurler does—even with the same API keys.

I wonder if it is some kind of location-based thing? My production server is in the US, but I dev from Australia.
